tests/integration/test_database/test_models/test_job.py

```diff
@@ -0,0 +1,32 @@
+from tests.integration.test_database.test_models import \
+    IntegrationTestCaseWithService
+from topchef.database.models import Job
+from topchef.database.schemas import JobStatus
+from uuid import uuid4
+
+
+class IntegrationTestCaseWithJob(IntegrationTestCaseWithService):
+    """
+    Base class for unit tests that have a job in them
+    """
+    @classmethod
+    def setUpClass(cls):
+        IntegrationTestCaseWithService.setUpClass()
+        cls.job_id = uuid4()
+        cls.job_status = JobStatus.WORKING
+        cls.job = Job(cls.job_id, cls.job_status,
+                      cls.valid_job_registration, cls.service,
+                      cls.valid_result)
+
+        cls.session.add(cls.job)
+        cls.session.commit()
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.session.delete(cls.job)
```

Comment:
Document this. Also, in the documentation, explain the postgres testing deadlock, as it is something to look out for when running Travis tests. The documentation for this method should state that any errant sessions should be committed prior to this method being called, or postgres will deadlock.